fabric-api-lookup-api-v1/README.md

```diff
@@ -0,0 +1,28 @@
+#  Fabric Api Lookup API (v1)
```

Comment:
I'm so sorry but I honestly think I need to commit bikeshed and say argue that "Api Lookup Api" is a terrible name that, while being very literal, is clunky to use and ends up hurting ease of understanding of the code due to the redundancy of the term "api". Calling it a component API isn't quite accurate either, since entity-component systems are inherently about managing storage as well as lookup. I think the best term to use here would be "property", as it's looking or traits of an object without caring about how they're stored.

Reply:
You may want to comment on #1234 about that, as it's nearing merge. This PR includes the commits of #1234 as well, which is why it appears in the diff... :/

Reply:
ahhhh, okay, I'll send it there